modules/apps/foundation/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_tag_selector.scss

```diff
@@ -22,6 +22,7 @@
 	}
 
 	.textboxlist-input-container {
+		float:none;
```

Comment:
I do not think we can do just a ```float: none;``` in making this change, since it would be odd to have a textarea following a bunch of listed tags.  I think we would have to rework the AUI component a little to get it to work right.